src/gui/categoryfilterwidget.h

```diff
@@ -26,7 +26,23 @@
  * exception statement from your version.
  */
 
- #include <QTreeView>
+#include <QSortFilterProxyModel>
+#include <QTreeView>
+
+class CategoryFilterProxyModel: public QSortFilterProxyModel
+{
+    Q_OBJECT
```

Comment:
@sledgehammer999: You guessed correctly.  And since `isSpecialItem()` is only available in `categoryfilterwidget.cpp`, it's probably easier to leave things where they are.